examples/apfl_example/server.py

```diff
@@ -68,6 +68,7 @@ def main(config: Dict[str, Any]) -> None:
     )
 
     server.metrics_reporter.dump()
```

Comment:
Maybe we can put this in the backlog. It feels like we can put the metrics_reporter dump in the server.shutdown() routine?